ros_buildfarm/templates/snippet/publisher_groovy-postbuild_reconfigure-updated-repositories.xml.em

```diff
@@ -0,0 +1,48 @@
+@(SNIPPET(
+    'publisher_groovy-postbuild',
+    script='\n'.join([
+        '// FIND UPDATED REPOSITORIES AND TRIGGER RECONFIGURE',
+        'import hudson.model.Cause.UserIdCause',
+        'import hudson.model.ParametersAction',
+        'import hudson.model.StringParameterValue',
+        'import java.io.BufferedReader',
+        'import java.util.regex.Matcher',
+        'import java.util.regex.Pattern',
+        'import jenkins.model.Jenkins',
+        '',
+        'reconfigure_repo_names = []',
+    ] +
+    ['reconfigure_repo_names << "%s"' % name for name in reconfigure_repo_names] +
+    [
+        '',
+        'pattern = Pattern.compile(".*- updated \'%s\' entry for repository \'(.+)\'")' % entry_type,
```

Comment:
`rosdistro` already supports this output: https://github.com/ros-infrastructure/rosdistro/blob/e9d6700d310e4d446a09506ef6a438659f1ce5d9/src/rosdistro/distribution_cache.py#L102